tgui/packages/tgui/interfaces/Uplink.js

```diff
@@ -134,7 +134,17 @@ const ItemList = (props, context) => {
   return items.map((item) => (
     <Section
       key={item.name}
-      title={item.name}
+      title={
+        item.is_illegal ? (
+          <Tooltip content="The product(s) is implemented with our best technology. Please do not let Nanotrasen R&D steal our confidentials.">
```

Comment:
```suggestion
          <Tooltip content="This product is powered by our latest technology. Please do not let Nanotrasen R&D steal our confidential designs.">
```
I'd go with a single item here, you can't really mix plural and singular effectively in English, some additional grammatical corrections. Using latest just sounds better imo.